app/spec/controllers/cbv/applicant_informations_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe Cbv::ApplicantInformationsController, type: :controller do
+  describe "#show" do
+    let(:current_time) { Time.now }
+    let(:cbv_flow) do
+      create(:cbv_flow,
+        created_at: current_time
+      )
+    end
+
+    before do
+      session[:cbv_flow_id] = cbv_flow.id
+    end
+
+    context "when rendering views" do
+      render_views
+
+      it "renders the sandbox fields" do
+        get :show
+
+        puts "response.status: #{response.status}"
+
+        expect(response.body).to include("first_name")
+        expect(response.body).to include("middle_name")
+        expect(response.body).to include("last_name")
+        expect(response.body).to include("case_number")
+      end
+
+      it "stays on the same page and shows the right errors when required fields are missing" do
+        post :update, params: {
+          cbv_applicant_sandbox: {
+            cbv_applicant: {
+              first_name: "Tim", # required
+              middle_name: "",
+              last_name: "", # required
+              case_number: "" # required
+            }
+          }
+        }
+
+        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response.body).not_to include(I18n.t(".cbv.applicant_informations.sandbox.fields.first_name.blank"))
+        expect(response.body).not_to include(I18n.t(".cbv.applicant_informations.sandbox.fields.middle_name.blank"))
+        expect(response.body).to include(I18n.t(".cbv.applicant_informations.sandbox.fields.last_name.blank"))
+        expect(response.body).to include(I18n.t(".cbv.applicant_informations.sandbox.fields.case_number.blank"))
```

Comment:
The beginning dot isn't doing anything in here. This is because a dot (relative I18n key) is [a nice feature](https://guides.rubyonrails.org/i18n.html#lazy-lookup) of the view helper method `translate` (aka `t`). It knows which view or controller you're in and prepends it to the key before calling `I18n.t`. But since we're here in tests, calling `I18n.t` directly, whatever raw key we pass in will be used directly.

```suggestion
        expect(response.body).not_to include(I18n.t("cbv.applicant_informations.sandbox.fields.first_name.blank"))
        expect(response.body).not_to include(I18n.t("cbv.applicant_informations.sandbox.fields.middle_name.blank"))
        expect(response.body).to include(I18n.t("cbv.applicant_informations.sandbox.fields.last_name.blank"))
        expect(response.body).to include(I18n.t("cbv.applicant_informations.sandbox.fields.case_number.blank"))
```